client/src/ui/molecules/language-toggle/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Translation } from "../../../document/types";
+import { useGA } from "../../../ga-context";
+
+import "./index.scss";
+
+export function LanguageToggle({
+  locale,
+  translations,
+}: {
+  locale: string;
+  translations: Translation[];
+}) {
+  const ga = useGA();
+
+  function getLink(locale) {
+    for (const translation of translations) {
+      if (translation.locale.toLowerCase() === "en-us") {
+        return (
+          <a
+            href={translation.url}
+            onClick={() => {
+              ga("send", {
```

Comment:
This will almost never send.

Reply:
The browser will cancel all operations and switch to the new URL.

Reply:
Ah yes, would have had to cancel the default event.